crates/router/src/configs/settings.rs

```diff
@@ -242,6 +243,15 @@ pub struct DummyConnector {
     pub discord_invite_url: String,
 }
 
+#[derive(Debug, Deserialize, Clone)]
+pub struct CorsSettings {
+    #[serde(deserialize_with = "deserialize_hashset")]
+    pub origins: HashSet<String>,
+    pub max_age: usize,
+    #[serde(deserialize_with = "deserialize_hashset")]
+    pub allowed_methods: HashSet<String>,
```

Comment:
Include `origins` and `allowed_methods` in `with_list_parse_key()`.

https://github.com/juspay/hyperswitch/blob/774a0322aa4b36d87b122e47cd893383e262de12/crates/router/src/configs/settings.rs#L675-L679